nss-day-cohort-48/rare-api-orchestra-outlaws
# Mec posts

#### Changes Made

1. Added `post.py` to `models` directory.
2. Added 'post.py' to 'views' directory.
3. Modified `urls.py` to register path to '/posts'

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout mec-posts
   ```
2. makemigrations
3. load users, rareusers, tokens, categories, and posts data
4. In Postman, the following HTTP methods should all function at http://localhost:8000/posts:
     GET, POST, PUT, DELETE
5. You should also be able to GET a single post at /posts/{post.id}
6. You should also be able to GET all the posts by a single user at /posts?rare_user={rare_user.id}
